.github/workflows/deploy.yml

```diff
@@ -176,3 +176,36 @@ jobs:
       project_type: ui
       azure_app_name: ${{ matrix.azure_app_name }}
     secrets: inherit
+
+  run-api-acceptance-tests:
+    if: inputs.environment == 'Test' || inputs.environment == 'Test2'
```

Comment:
Tiny nitpick, but there's a builtin `contains` function (if you want to use it)

I believe it would be `if: contains(inputs.environment, 'Test')`

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#contains

Reply:
Should actually match exactly not contains or starts with as these are environment variables. What is the advantage of using the contains or starts with?

Reply:
Just conciseness, and would mean adding the integration test env in the future would be automatically picked up. Not fussed either way tho